profiles/models.py

```diff
@@ -69,6 +69,7 @@ class Profile(models.Model):
     banner_image = models.ImageField(validators=[validate_image_format, validate_image_size], null=True)
 
     is_deleted = models.BooleanField(default=False)
+    is_approved = models.BooleanField(default=False)
```

Comment:
What is a meaning of this field? I'm not aware about all updates of requirements, do we have profile approval included?

Reply:
At the meeting, we discussed this field. That is, the administrator checks the EDRPOU and if it is valid, then sets is_approved=True.
We don't have exact requirements for the admin panel.
If this field is not okay, I can remove it from the code.